mlxtend/feature_selection/exhaustive_feature_selector.py

```diff
@@ -24,6 +24,42 @@
 from ..externals.name_estimators import _name_estimators
 
 
+def _merge_lists(nested_list, high_level_indices=None):
+    """
+    merge elements of lists (of a nested_list) into one single list
+
+    Parameters
+    ----------
+    nested_list: List
+        a  list whose elements must be list as well.
+
+    high_level_indices: list or tuple, default None
```

Comment:
I think the function would work automatically for both, right? I mean if we would restrict it to one type, that would actually add extra code? I'd say since that's an internal, private function, don't worry about it 😅